README.md

```diff
@@ -73,5 +73,5 @@ This was the subject of a talk that was given @ the Linux Plumbers Conference
 http://linuxplumbersconf.com/2017/ocw//system/presentations/4818/original/TPM2-kernel-evnet-app_tricca-sakkinen.pdf
 
 # Related Specifications
-* [TPM2 Software Stack Access Broker and Resource Manager](https://trustedcomputinggroup.org/wp-content/uploads/TSS_TCTI_v1.0_r04_Public-Review.pdf)
+* [TPM2 Software Stack Access Broker and Resource Manager](https://www.trustedcomputinggroup.org/wp-content/uploads/TSS-TAB-and-Resource-Manager-00-91-PublicReview.pdf)
```

Comment:
This link works which is a step in the right direction. But it's to a very old version. The latest version is here: https://trustedcomputinggroup.org/wp-content/uploads/TSS-TAB-and-Resource-Manager-ver1.0-rev16_Public_Review.pdf